### ISSUE TITLE
Olevba: officepurge bug

### ISSUE THREAD

**Issue by Solumnant** [decalage2/oletools#645]
**Affected tool:**
olevba

**Describe the bug**
Olevba errors out when run on files cleaned by officepurge https://github.com/fireeye/OfficePurge

**File/Malware sample to reproduce the bug**
https://github.com/fireeye/OfficePurge/tree/main/sample-data 

**How To Reproduce the bug**
Run olevba on any of the _PURGE files

**Expected behavior**
olevba extracts the macros in the file without erroring out

**Console output / Screenshots**

```
 F:\downloads> python.exe C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py .\word_calc_PURGED.doc
olevba 0.56 on Python 3.8.6 - http://decalage.info/python/oletools
===============================================================================
FILE: .\word_calc_PURGED.doc
Type: OLE
ERROR    VBA code returned by extract_all_macros is not a string
ERROR    VBA code returned by extract_all_macros is not a string
Traceback (most recent call last):
  File "C:\users\ben\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 4029, in process_file
    vba_code_filtered = filter_vba(vba_code)
  File "C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 2133, in filter_vba
    vba_lines = vba_code.splitlines()
AttributeError: 'NoneType' object has no attribute 'splitlines'
ERROR    Error processing file .\word_calc_PURGED.doc ('NoneType' object has no attribute 'splitlines')!
Traceback (most recent call last):
  File "C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 4029, in process_file
    vba_code_filtered = filter_vba(vba_code)
  File "C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 2133, in filter_vba
    vba_lines = vba_code.splitlines()
AttributeError: 'NoneType' object has no attribute 'splitlines'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 4357, in process_file
    vba_parser.process_file(show_decoded_strings=options.show_decoded_strings,
  File "C:\users\solumnant\appdata\local\packages\pythonsoftwarefoundation.python.3.8_qbz5n2kfra8p0\localcache\local-packages\python38\site-packages\oletools\olevba.py", line 4110, in process_file
    raise ProcessingError(self.filename, exc)
ProcessingError: Error processing file .\word_calc_PURGED.doc ('NoneType' object has no attribute 'splitlines')
```

**Version information:**
 - OS: Windows/Linux
 - Python version: 3.8.6
 - oletools version: .56

**Comment by Solumnant** [comment]
This issue was fixed in 0.56.1.dev2 and is mentioned in #629 . The resultant bug with the CompressedChunkSignature has already been raised in #619

**Comment by decalage2** [comment]
Thank you, this is a good reminder that I need to release oletools 0.56.1. ;-)
